napari/components/_tests/test_layer_slicer.py

```diff
@@ -384,6 +360,8 @@ def test_submit_with_one_3d_points(layer_slicer):
         future = layer_slicer.submit(layers=[layer], dims=dims)
         assert not future.done()
 
+    _wait_for_result(future)[layer]
+
```

Comment:
Should we have an assert here? We wait for it to get done, but don't check anything after it finishes.